migrate/20250212_add_provider_locations.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    create_enum(:provider_name, ["hetzner", "leaseweb", "latitude", "global"])
+    create_table(:provider_location) do
+      column :provider_name, :provider_name, null: false
+      column :display_name, String, null: false
+      column :internal_name, String, null: false
+      column :ui_name, String, null: false
+      column :visible, :boolean, null: false
+      column :id, :uuid, primary_key: true
```

Comment:
nit: curious about the reason for placing the `id` at the end, rather then the top

Reply:
Not a single reason, I just started typing the columns that I need and added the id later :D

Reply:
Unless there is a good reason to deviate, for consistency with other tables, `id` should be the first column.

Reply:
I moved it